oracle/updaterset.go

```diff
@@ -10,7 +10,7 @@ import (
 
 func UpdaterSet(_ context.Context) (driver.UpdaterSet, error) {
 	us := driver.NewUpdaterSet()
-	for year, lim := 2007, time.Now().Year(); year != lim; year++ {
+	for year, lim := 2007, time.Now().Year(); year <= lim; year++ {
```

Comment:
From what I can tell, let's say it's January 1, 2026 and the 2026 OVAL file does not exist yet, we will still create the UpderSet, no problem, but we may see error messages for who knows how long. Is that ok?

Reply:
I don't know enough about UpdaterSets to answer that. If we've solved this problem elsewhere, I'd be in favor of that solution for consistency. If we haven't, I need to better understand the intention of the failure mode of an UpdaterSet. My gut reaction is to log it as a warning. Or maybe even introduce a new error that indicates a 404, warn on that, and error in all other cases.

Reply:
The change looks good to me. Typically, if we encounter some error messages but the overall Scanner V4 update workflow still succeeds, we rarely investigate individual errors or warnings unless the entire workflow fails.
It appears that in the Oracle update process, the libvuln update manager runs each updater independently. Even if one or more Oracle updaters fail, the rest continue executing.

Reply:
Yep, confirming what @daynewlee said, the update process will carry on and errors will be aggregated and logged at the end.